invenio_rdm_records/fixtures/tasks.py

```diff
@@ -43,9 +46,19 @@ def get_authenticated_identity(user_id):
 
 @shared_task
 def create_vocabulary_record(service_str, data):
-    """Create a vocabulary record."""
+    """Create or update a vocabulary record."""
     service = current_service_registry.get(service_str)
-    service.create(system_identity, data)
+    pid_type = data['type']
+    pid_value = data['id']
+    try: 
+        pid = (pid_type,pid_value)
+        # If the entry hasn't been added, this will fail
+        record = Vocabulary.pid.resolve(pid)
+        service.update(system_identity,pid, data=data)
```

Comment:
minor: this might "re-activate" deleted entries, but I guess we assume that if they are in the data file and one runs the command we *want to reactive it*. So I'd say all good.